src/serviceorchestration/5_0_0/sd/service-orchestration/service_orchestration_sd.tex

```diff
@@ -160,7 +160,7 @@ \subsection{Access policy}
 \section{Service Operations}
```

Comment:
To every subsection: Please copy here what the appropriate operation does, Jerker wanted this. For example:
`
Operation \textit{pull}  performs the orchestration process and returns the matching service instances. The behavior of this operation...`